modules/virt-monitoring-vm-migration-web.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
```

Comment:
Pan, if we get the web console overview PR merged in time, I think these short UI procedures could be replaced with a link, in the assembly, to the appropriate section. That would save us a lot of time. :-) Just a thought.